### PR TITLE
Add source maps upload instructions for CRA

### PR DESCRIPTION
**Pull request by lforst**
Resolves https://github.com/getsentry/sentry-docs/issues/7106

Adds instructions on how to get source maps working with Create React App.

### CODE REVIEW

File: src/includes/sourcemaps-create-react-app.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+### 5. Deploy your Application
+
+If you're following this guide from your local machine, then you've successfully:
+
+1. Generated minified source and source map files (artifacts) by running your application's build process
+2. Injected Debug IDs into the artifacts you've just generated
+3. Uploaded those artifacts to Sentry with our upload command
+
+The last step is deploying a new version of your application using the generated artifacts you created in step one.
+**We strongly recommend running all of the things you have configured above to run inside your CI/CD Pipeline**, to ensure each subsequent deploy will have readable stack traces in Sentry error events.
```
Comment (vivianyentran) [inline src/includes/sourcemaps-create-react-app.mdx:114]:
```suggestion
**We strongly recommend running everything you've configured above inside your CI/CD pipeline** to ensure each subsequent deploy will have readable stack traces in Sentry error events.
```

File: src/includes/sourcemaps-create-react-app.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+### 5. Deploy your Application
+
+If you're following this guide from your local machine, then you've successfully:
+
+1. Generated minified source and source map files (artifacts) by running your application's build process
+2. Injected Debug IDs into the artifacts you've just generated
+3. Uploaded those artifacts to Sentry with our upload command
+
+The last step is deploying a new version of your application using the generated artifacts you created in step one.
```
Comment (vivianyentran) [inline src/includes/sourcemaps-create-react-app.mdx:113]:
```suggestion
The last step is deploying a new version of your application using the generated artifacts you created.
```

File: src/includes/sourcemaps-create-react-app.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+```bash
+sentry-cli sourcemaps upload /path/to/build
+```
+
+#### Verify That Artifact Bundles Were Uploaded
+
+Open up Sentry and navigate to **Project Settings > Source Maps**.
+If you choose “Artifact Bundles” in the tabbed navigation, you'll see all the artifact bundles that have been successfully uploaded to Sentry.
+
+### 5. Deploy your Application
```
Comment (vivianyentran) [inline src/includes/sourcemaps-create-react-app.mdx:105]:
```suggestion
### 5. Deploy Your Application
```

**Review by vivianyentran — APPROVED** [review]
Added some small edits but otherwise LGTM! Thanks for adding these docs!

**Comment by smeubank** [comment]
lgtm! just added it to the overview page under guides

image

File: src/includes/sourcemaps-create-react-app.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+SENTRY_PROJECT=___PROJECT_SLUG___
+# Auth tokens can be obtained from
+# https://sentry.io/settings/auth-tokens/
+SENTRY_AUTH_TOKEN=___ORG_AUTH_TOKEN___
+```
+
+### 3. Inject Debug IDs Into Build Artifacts
+
+Debug IDs are used to match the stack frame of an event with its corresponding minified source and source map file.
+Visit [What are Artifact Bundles](/platforms/javascript/sourcemaps/troubleshooting_js/artifact-bundles/) if you want to learn more about Artifact Bundles and Debug IDs.
```
[the remark this answers is not in the document]
Comment (lforst) [reply src/includes/sourcemaps-create-react-app.mdx:61]: "Artifact Bundles" and "Debug IDs" are terms we invented which is why we capitalize them. If we lowercase them here we need to do this over the entire docs.

File: src/includes/sourcemaps-create-react-app.mdx (hunk 2 above)
Comment (lforst) [reply src/includes/sourcemaps-create-react-app.mdx:113]: This is critical

**Comment by lforst** [comment]
Thanks @vivianyentran for the review!

**Pull request merged**